src/sentry/static/sentry/app/views/organizationDiscover/sidebar/queryFields.jsx

```diff
@@ -138,8 +150,17 @@ export default class QueryFields extends React.Component {
             disabled={isLoading}
           />
         </Fieldset>
+        <FieldSeparator />
         <Fieldset>{actions}</Fieldset>
-      </div>
+        <DocsSeparator />
+        <TextLink href="https://docs.sentry.io/product/discover/">
```

Comment:
We generally use the ExternalLink component for opening links outside the app, it opens in new window which we'd want as well

Reply:
Added it! It does turn blue when you hover over it, -- is that okay? cc: @Chrissy